docs/content/en/tasks/data-plane-registration/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Data plane registration"
+linkTitle: "Data Plane Registration"
+description: "Managing connections between multiple Kubernetes clusters in a multi cluster environment."
+---
+
+## Background
+
+Details of the multi-cluster connectivity architecture can be found [here]({{< relref "reference/multi-cluster" >}}). Details regarding ClientConfigs, ServiceAccounts, and Kubeconfig secrets are dealt with there.
+
+Proceed with caution before deleting a ClientConfig or the secrets backing it. If there are any K8ssandraClusters that use the kube config provided by the ClientConfig, then the operator won't be able to properly manage them.
+
+#### Creating a ClientConfig and Kubeconfig secret
+Registering a data plane to a control plane can be error prone if done by hand. Instead use the `k8ssandra-client` CLI tool which can be found [here](https://github.com/k8ssandra/k8ssandra-client). It can be installed as a kubectl plugin by placing it on your `$PATH` or run completely separately.
```

Comment:
**Suggestion:**
```suggestion
Registering a data plane to a control plane can be error prone if done by hand. Instead use the `k8ssandra-client` CLI tool which can be found [here](https://github.com/k8ssandra/k8ssandra-client). It can be installed as a kubectl plugin by placing it on your `$PATH` (invoked with `kubectl k8ssandra ...`) or run completely separately (invoked with `./kubectl-k8ssandra ...`).
```

Reply:
This one is definitely better, accepting.